app/javascript/packs/modCenter.jsx

```diff
@@ -5,9 +5,16 @@ let elementLoaded = false;
 
 function loadElement() {
   const root = document.getElementById('mod-index-list');
+  const isMobileDevice = typeof window.orientation !== 'undefined';
+
   if (root) {
     render(<ModerationArticles />, root);
   }
+
+  if (isMobileDevice) {
+    // eslint-disable-next-line no-alert
+    alert('The Mod Center is best viewed on desktop');
```

Comment:
Consider using the snackbar or something that is not an `alert`. cc: @ludwiczakpawel

Reply:
great point Nick, thanks. Is there a way to have the snackbar NOT fade out until `Dismiss` is clicked/tapped?

Reply:
+1 on the snackbar

Reply:
@msarit that would be great, by reading the code it seems like the only way to hack that currently is to have a super long lifespan

Reply:
If it's something you want to persist for longer than 3 seconds, maybe something else would make more sense like a [notice](https://storybook.dev.to/?path=/story/3-components-notices-html--warning).

Reply:
Added snackbar, with a `lifespan` set to 0. That seemed to make it so it doesn't disappear on its own.